front/src/components/mmt-locale-switcher/mmt-locale-switcher.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="locale-switcher">
+  <v-select :items="locales"
+            v-on:change="switchLocale"
+            :label="$t('footer.language')"
```

Comment:
why language is in part of footer? it can be comfused. we can make common section without any view or component part name in translations

Reply:
today switcher is in footer, tomorrow it will be placed into menu. translation won\t be changed, but we had to remove it to menu section in translation. so, common section or translation without section would be better for some words

Reply:
Replaced with 'locale.language'